metadata.Dockerfile

```
@@ -12,7 +12,7 @@ ENTRYPOINT ["sh", "-o", "pipefail", "-c"]
 FROM base AS latest-kubectl-version
 
 CMD [ \
-    "curl -fs https://storage.googleapis.com/kubernetes-release/release/stable.txt | \
+    "curl -fs https://dl.k8s.io/release/stable.txt | \
```

```suggestion
    "curl -fsL https://dl.k8s.io/release/stable.txt | \
```
How about adding the -L option to follow redirects? It seems that the current URL returns a 302 status when using curl.

```sh
$ curl -fs https://dl.k8s.io/release/stable.txt
<html>
<head><title>302 Found</title></head>
<body>
<center><h1>302 Found</h1></center>
<hr><center>nginx</center>
</body>
</html>
```